text/0000-trajectory-builder-interface.md

```diff
@@ -0,0 +1,31 @@
+# Introduction of Unified TrajectoryBuilderInterface.
```

Comment:
Unify to a Single TrajectoryBuilderInterface? (You don't introduce something new.)